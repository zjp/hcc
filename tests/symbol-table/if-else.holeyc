int a;
bool b;
void f() {
	TOCONSOLE "AAAAAAAAAAAA";
}
void g() {
	TOCONSOLE "BBBBBBBBBBBB";
}
int main(int c) {
	if(b) {
		f();
	} else {
		g();
	}
}
