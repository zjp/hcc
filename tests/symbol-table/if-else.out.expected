int a;
bool b;
void f(){
	TOCONSOLE "AAAAAAAAAAAA";
}
void g(){
	TOCONSOLE "BBBBBBBBBBBB";
}
int main(int a){
	if (b(bool)){
		f(->void)();
	} else {
		g(->void)();
	}
}
